reledmac.dtx

```diff
@@ -3191,7 +3249,7 @@
 % Thats is called \enquote{antilabe}.
 % In this case, the typesetting of the second half  must start at the horizontal position where the typesetting of the first half stops.
 %
-% \Reledmac\  provides tool for such problematic, but you must enable them with the \opt{antilabe} option when loading package, as this option is quite ressource consuming.
+% \Reledmac\  provides tool for such problematic, but you must enable them with the \option{antilabe} option when loading package, as this option is quite ressource consuming.
```

Comment:
"\Reledmac\  provides tool for such problematic" → "\Reledmac\ provides tools to cope with this problem"

Reply:
done, and I correct also to "as these tools are quite ressource consuming"